roles/kube_proxy/files/kube-proxy-images.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStreamTag
+metadata:
+  name: node:v3.10
```

Comment:
Its best not to hardcode version instead something like this should do the trick s/v3.10/${version}/

Reply:
This is a name of the imagestream, it needs to be hardcoded so that an upgraded cluster would have several imagestreams for major versions

Reply:
Yeah, for a PR against 'git master' I would expect the tag to be different, but I was just copying what some of the existing ImageStream tags already were thus "v3.10"